frontend/scripts/e2e-server.js

```diff
@@ -1,10 +1,14 @@
 import express from "express";
+import compression from "compression";
```

Comment:
Q: Shouldn't `package.json` be updated with this as a dev dependency?

Reply:
Yes!, thank you very much! :)